__tests__/app.tests.js

```diff
@@ -242,7 +279,7 @@ describe("app", () => {
           .then(({ body }) => {
             expect(body.msg).toBe("Article does not exist");
           });
```

Comment:
Hey Wes. For number 11, can we have a new test to check for comment count, rather than amending one of the old ones please? Thanks!